### ISSUE TITLE
Documentation for model class properties

### ISSUE THREAD

**Issue by sstainba** [mentalstack/yolov5-net#69]
For those of us who are newer to this, could you provide some more documentation on the YoloModel class and how the properties we need to override relate to our model?  Perhaps show an example input/output tensor and then which values to use for which properties?

Specifically, the Dimensions, Strides, Anchors, Shapes and Confidence values.

**Comment by Red007Master** [comment]
Well you need set dimension to {Lables.count}+5 to make it work. (all i can really help with) but most of values (Strides, Anchors, Shapes) it's related to training process (i believe so but Anchors 100% related)
(I made fork with model save/load to/from json functionality if you need it: github.com/Red007Master/yolov5-net-SaveLoadModels)
If you have and questions i glad to help(if i can).

**Comment by Red007Master** [comment]
> For those of us who are newer to this, could you provide some more documentation on the YoloModel class and how the properties we need to override relate to our model? Perhaps show an example input/output tensor and then which values to use for which properties?
> 
> Specifically, the Dimensions, Strides, Anchors, Shapes and Confidence values.

Well you need set dimension to {Lables.count}+5 to make it work. (all i can really help with) but most of values (Strides, Anchors, Shapes) related to training process (i believe so but sure about Anchors)
(I made fork with model save/load to/from json functionality if you need it: github.com/Red007Master/yolov5-net-SaveLoadModels)
If you have and questions i glad to help(if i can).

**Comment by sstainba** [comment]
Yeah, I've learned that in the past week or so.  The anchors come from the yolo config files but I'm still not sure about strides (which all seem to be base-2 with the same count as the anchors) or the shapes.

I do think this project needs a little love - specifically some abstraction into interfaces and maybe re-work to make it configurable at runtime instead of a generic needing a the inherited type.  I might get around to that at some point.
